src/NinjaTools.FluentMockServer.API/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using NinjaTools.FluentMockServer.API.Middleware;
+using HttpResponse = NinjaTools.FluentMockServer.Models.HttpEntities.HttpResponse;
+
+namespace NinjaTools.FluentMockServer.API.Extensions
+{
+    public static class ApplicationBuilderExtensions
+    {
+        public static void RunProxy(this IApplicationBuilder app, HandleProxyRequest handleProxyRequest)
+        {
+            if (app == null)
+            {
+                throw new ArgumentNullException(nameof(app));
+            }
+
+            if (handleProxyRequest == null)
+            {
+                throw new ArgumentNullException(nameof(handleProxyRequest));
+            }
+
+           // app.UseMiddleware<ProxyMiddleware<HandleProxyRequestWrapper>>(new HandleProxyRequestWrapper(handleProxyRequest))
+        }
+        
+        private class HandleProxyRequestWrapper : IProxyHandler
+        {
+            private readonly HandleProxyRequest _handleProxyRequest;
+
+            public HandleProxyRequestWrapper(HandleProxyRequest handleProxyRequest)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private constructor 'HandleProxyRequestWrapper'.](https://app.codacy.com/app/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4722826)